Prac_07/KivyDemos-master/squaring.kv

```diff
@@ -1,15 +1,20 @@
 BoxLayout:
     orientation: 'vertical'
-    TextInput:
-        id: input_number
-        text: '7'
-        font_size: 48
-        multiline: False
-    Button:
-        text: 'Square'
-        # the following line specifies the function in the app class to call when the button is pressed
-        on_press: app.handle_calculate(int(input_number.text))
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(int(input_number.text))
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 254, 1, 154)
```

Comment:
Color is a tricky one, what i discovered is that rgba tuples can use values between 1 and 0, a fraction out of 255.
In other words a hot pink (254, 1, 154) would look like (254/255, 1/255, 154/255, 1)